src/main/java/Task.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Creates a task with a description which is to be added
+ * to the task list.
+ */
+public class Task {
+
+	protected String description;
+	protected boolean isDone;
+
+	public Task(String description) {
+
+		this.description = description;
+		this.isDone = false;
+	}
+
+	/**
+	 * Returns a icon to let the user know that
+	 * the tasks has been labelled 'done'.
+	 *
+	 * @return An icon.
+	 */
+	public String getStatusIcon() {
+
+		return (isDone ? "x" : " "); //return tick or X symbols
+	}
+
+	/**
+	 * Changes the boolean from false to true to mark
+	 * the task as completed.
+	 */
+	public void markAsDone() {
+
+		this.isDone = true;
+	}
+
+
+	public void markNotDone(){
+
+		this.isDone = false;
+	}
+
+
+	/**
+	 * Returns the description of the task.
+	 *
+	 * @return Description of the task.
+	 */
+	public String getDescription() {
+
+		return this.description;
+	}
+
+	/**
+	 * Returns an integer 1 or 0 to indicate
+	 * whether the task is completed when the task list is
+	 * being written to a text document.
+	 *
+	 * @return 1 or 0 to indicate completion of task.
+	 */
+	public int getStatus() {
```

Comment:
This should really be a boolean... the encoding should be left to the class that actually does the serialisation